### ISSUE TITLE
Does the “total_steps” parameter mean The step within one epoch or across the one fitting session call?

### ISSUE THREAD

**Issue by neilteng** [tensorflow/addons#1704]
I just have some confusion about the parameter “total_steps”.

Does the “total_steps” parameter mean The step within one epoch or across the one fitting session call?

If it is the first case, what if the training step in one epoch is less than the “total_steps” parameter?
If it is the second case, what if it excesses the “total_steps”?

BTW, do you think with RAdam, whether we still need a learning rate schedule?
Thank you for your help.

**Comment by WindQAQ** [comment]
1. The second case. Actually, it is used to measure how many warmup steps the optimizer needs so it won't cause anything dangerous if current steps (`self.iterations`) is larger than `total_steps`.

2. It depends. We cannot answer this kind of question.

Feel free to reopen the issue if you have further question. Thank you!
